Library/ENV/scm/git

```diff
@@ -10,8 +10,8 @@ fi
 exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
 #!/usr/bin/env ruby -W0
 
-# This script because we support $GIT, $HOMEBREW_SVN, etc. and Xcode-only
-# configurations. Order is careful to be what the user would want.
+# This script because we support $GIT, $HOMEBREW_SVN, etc., Xcode-only and
```

Comment:
Perhaps we should stop supporting `GIT`? It seems to cause more problems than it solves (and leave `HOMEBREW_GIT`)